Ginger/Ginger/AutomatePageLib/AddActionMenu/SharedRepositoryLib/SharedRepositoryOperations.cs

```diff
@@ -276,21 +276,22 @@ public static RepositoryItemBase GetMatchingRepoItem(RepositoryItemBase item, IE
 
         public static bool CheckIfSureDoingChange(RepositoryItemBase item, string changeType)
         {
-            RepositoryItemUsagePage usagePage = null;
-            usagePage = new RepositoryItemUsagePage(item, true);
-            if (usagePage.RepoItemUsages.Count > 0)//TODO: check if only one instance exist for showing the pop up for better performance
+            //RepositoryItemUsagePage usagePage = null;
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5970337)